raiden-ts/package.json

```diff
@@ -74,7 +76,6 @@
     "vuepress": "^1.2.0"
   },
   "dependencies": {
-    "@types/lodash": "^4.14.149",
```

Comment:
I moved this to devDependencies unless someone has a good reason for it to be in dependencies. I saw that this package also takes up quite a big of space in the dApp bundle. My assumption is that it's bundled into the dApp because it's not a dev dep